### PR TITLE
Fix #863: Lowfi Landscape ProfileRenameActivity

### LINKED ISSUE #863: Lowfi landscape ProfileRenameActivity
Mock: https://xd.adobe.com/view/ee9e607b-dbd6-4372-48dc-b687d32af3da-98af/screen/b7447549-a7c3-48a9-8ad2-876d0a347708/RG-MN-Register-Empty-36

Added test cases for landscape mod along with implementation.

### PR DESCRIPTION
**Pull request by rt4914**
## Explanation

Fixes #863 

Mock Portrait: https://xd.adobe.com/view/e8aa4198-3940-47f9-514a-f41cc54457f6-9e9b/screen/d0227921-1c24-4bb7-8667-a8fa5f4742dd/AC-EP-Rename-

Mock Landscape: https://xd.adobe.com/view/ee9e607b-dbd6-4372-48dc-b687d32af3da-98af/screen/b7447549-a7c3-48a9-8ad2-876d0a347708/RG-MN-Register-Empty-36

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: app/src/main/java/org/oppia/app/settings/profile/ProfileRenameActivityPresenter.kt [hunk 1]
```diff
@@ -83,6 +91,21 @@ class ProfileRenameActivityPresenter @Inject constructor(
     }
   }
 
+  private fun addTextChangedListener(
+    profileInputView: ProfileInputView,
+    onTextChanged: (CharSequence?) -> Unit
+  ) {
+    profileInputView.addTextChangedListener(object : TextWatcher {
+      override fun onTextChanged(p0: CharSequence?, p1: Int, p2: Int, p3: Int) {
+
```
Comment (veena14cs) [inline app/src/main/java/org/oppia/app/settings/profile/ProfileRenameActivityPresenter.kt]: Remove this blank line.

File: app/src/main/java/org/oppia/app/settings/profile/ProfileRenameViewModel.kt [hunk 2]
```diff
@@ -8,5 +8,10 @@ import javax.inject.Inject
 /** The ViewModel for [ProfileRenameActivity]. */
 @ActivityScope
 class ProfileRenameViewModel @Inject constructor() : ObservableViewModel() {
+  val inputName = ObservableField("")
   val nameErrorMsg = ObservableField("")
+
+  fun sas(){
```
Comment (veena14cs) [inline app/src/main/java/org/oppia/app/settings/profile/ProfileRenameViewModel.kt]: Can you make this method name more contextual.

**Review by veena14cs — APPROVED** [review]
LGTM beside some nit change.

File: app/src/main/java/org/oppia/app/settings/profile/ProfileRenameActivityPresenter.kt (hunk 1 above)
Comment (rt4914) [reply app/src/main/java/org/oppia/app/settings/profile/ProfileRenameActivityPresenter.kt]: Done

File: app/src/main/java/org/oppia/app/settings/profile/ProfileRenameViewModel.kt (hunk 2 above)
Comment (rt4914) [reply app/src/main/java/org/oppia/app/settings/profile/ProfileRenameViewModel.kt]: Removed

**Pull request merged**